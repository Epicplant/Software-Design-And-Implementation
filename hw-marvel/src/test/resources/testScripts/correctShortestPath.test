# Tests shortest path for the complicated set provided by the assignment

# Loads a new graph from a csv file
LoadGraph heroes staffSuperheroes.csv

# Finds shortest path between heroes
FindPath heroes Notkin-of-the-Superhuman-Beard Grossman-the-Youngest-of-them-all