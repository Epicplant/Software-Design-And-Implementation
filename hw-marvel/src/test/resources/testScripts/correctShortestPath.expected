# Tests shortest path for the complicated set provided by the assignment

# Loads a new graph from a csv file
loaded graph heroes

# Finds shortest path between heroes
path from Notkin-of-the-Superhuman-Beard to Grossman-the-Youngest-of-them-all:
Notkin-of-the-Superhuman-Beard to Grossman-the-Youngest-of-them-all via CSE331
